eahub/localgroups/views.py

```diff
@@ -1,14 +1,38 @@
 import json
-from datetime import datetime 
+from datetime import datetime
 
 from django.urls import reverse_lazy
 from django.shortcuts import render, redirect, get_object_or_404
 from django.contrib.auth.decorators import login_required
-
+from django.core.mail import mail_managers
+from django.contrib.sites.shortcuts import get_current_site
+from django.contrib import messages
+from django.template.loader import render_to_string
 from .models import LocalGroup as Group
 
 def view_group(request, slug):
     group = get_object_or_404(Group, slug=slug)
+
     return render(request, 'eahub/group.html', {
         'group': group
     })
+
+@login_required(login_url=reverse_lazy('login'))
+def claim_group(request, slug):
+    group = get_object_or_404(Group, slug=slug)
+    if request.method == 'POST':
```

Comment:
As-is, this will raise a 500 error on a GET request, because you don't actually return a response in that case. [`require_POST`](https://docs.djangoproject.com/en/2.1/topics/http/decorators/#django.views.decorators.http.require_POST) is the right way to solve this problem.